controlplane/kubeadm/api/v1beta1/v1beta2_condition_consts.go

```diff
@@ -20,14 +20,33 @@ import clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
 
 // KubeadmControlPlane's Available condition and corresponding reasons that will be used in v1Beta2 API version.
 const (
-	// KubeadmControlPlaneAvailableV1Beta2Condition is True if the control plane can be reached, EtcdClusterHealthy is true,
-	// and CertificatesAvailable is true.
+	// KubeadmControlPlaneAvailableV1Beta2Condition is true if KubeadmControlPlane not deleted, `CertificatesAvailable` is true,
+	// at least one Machine with Kubernetes API server, scheduler and controller manager healthy,
```

Comment:
```suggestion
	// at least one Machine with healthy control plane components,
```
I think this would be better here. Otherwise we have an incomplete list of what we explain in more details below (etcd pod / member)